internal/cmd/launcher.go

```diff
@@ -76,22 +77,21 @@ func (l *Launcher) Launch() {
 	// if the command does not have dependencies or a custom build
 	if !customBuildRequired(build.Version, deps) {
 		l.gs.Logger.
-			Debug("The current k6 binary already satisfies all the required dependencies," +
-				" it isn't required to provision a new binary.")
+			Debug("Binary provisioning is not required to execute this command.")
```

Comment:
I find the previous message better because it explains _the why_. Can we restore it?